web/packages/teleterm/src/ui/Search/SearchBar.tsx

```diff
@@ -185,7 +184,9 @@ function SearchBar() {
 const Input = styled.input`
   height: 38px;
   width: 100%;
-  min-width: calc(${props => props.theme.space[9]}px * 2);
+  // min-width causes the filters and the actual input text to be broken into
+  // two lines when there is no space
+  min-width: calc(${props => props.theme.space[8]}px * 2);
```

Comment:
Yeah, when the width was 9, the content of the input was a little too wide.

>But at the smallest window width, it's probably better to make the top bar usable rather than to keep both filters in the same line.

👍